### PR TITLE
Feature/56

### PR DESCRIPTION
**Pull request by shivenduverma-sap**

### CODE REVIEW

**Comment by shivenduverma-sap** [comment]
TODO after review, before merge: 
  - use latest version of ocm-controller
  - use latest version of mpas-project-controller

File: Tiltfile [hunk 1]
```diff
@@ -88,10 +91,10 @@ docker_build_with_restart(
...
       './bin',
-      'hack/entrypoint.sh',
+      './hack/entrypoint.sh',
     ],
     live_update = [
         sync('./bin/manager', '/manager'),
         sync('./hack/entrypoint.sh', '/entrypoint.sh'),
     ],
-)
+)
```
Comment (Skarlso) [inline Tiltfile:100]: Do you know why your IDE removes the extra lines? They should not be changed.

File: go.mod [hunk 2]
```diff
@@ -11,6 +11,18 @@ replace github.com/google/go-containerregistry => github.com/google/go-container
...
+
+replace github.com/stretchr/testify => github.com/stretchr/testify v1.8.2
+
+replace sigs.k8s.io/controller-runtime v0.15.0 => sigs.k8s.io/controller-runtime v0.14.6
+
+replace k8s.io/client-go => k8s.io/client-go v0.26.3
+
+replace k8s.io/api => k8s.io/api v0.26.3
+
+replace github.com/stretchr/testify v1.8.3 => github.com/stretchr/testify v1.8.2
```
[the remark this answers is not in the document]
Comment (shivenduverma-sap) [reply go.mod:24]: Same as: https://github.com/open-component-model/MPAS/pull/68#discussion_r1289850515

File: controllers/productdeploymentpipeline_scheduler.go [hunk 3]
```diff
@@ -82,7 +82,7 @@ func (r *ProductDeploymentPipelineScheduler) Reconcile(ctx context.Context, req
 		}
 	}()
 
-	target, err := r.SelectTarget(ctx, obj.Spec.TargetRole, r.MpasSystemNamespace)
+	target, err := r.SelectTarget(ctx, obj.Spec.TargetRole, obj.Namespace)
```
Comment (Skarlso) [inline controllers/productdeploymentpipeline_scheduler.go:85]: adopting this change in https://github.com/open-component-model/mpas-product-controller/pull/35/files so Gerald can move forward with some demos.

**Review by Skarlso — APPROVED** [review]

**Pull request merged**